recipes/dbghaplo/meta.yaml

```
@@ -0,0 +1,47 @@
+  run_exports:
+    - {{ pin_subpackage('dbghaplo', max_pin="x.x") }}
+
+requirements:
+  build:
+    - {{ compiler("cxx") }}
+    - {{ compiler('rust') }} 
+    - cargo-bundle-licenses
+    - make
+    - cmake >=3.12
+  run:
+    - python 
```

yamllint

[error] trailing spaces

(trailing-spaces)